src/library/scala/concurrent/Future.scala

```diff
@@ -243,7 +243,12 @@ trait Future[+T] extends Awaitable[T] {
    *  this future. If this future is completed with an exception then the new
    *  future will also contain this exception.
    *
-   *  $forComprehensionExamples
+   *  Example:
+   *
+   *  {{{
+   *  val f = Future { "The future" }
+   *  val g = f map { (x: String) => x + " is now!" }
+   *  }}}
```

Comment:
While I agree that using the simple string interpolator would be a better practice I do not think it should be in this example.  I do not want to assume that somebody looking up how to use `map` also knows how to use the interpolator.   

I do like your removal of the extra parentheses though.